boards/b-l475e-iot01a/include/periph_conf.h

```diff
@@ -158,6 +164,44 @@ static const spi_conf_t spi_config[] = {
         .tx_dma_chan = 1,
         .rx_dma   = 0,
         .rx_dma_chan = 1,
+#endif
+    },
+    {
+        .dev      = SPI2,
+        .mosi_pin = GPIO_PIN(PORT_D, 4),
+        .miso_pin = GPIO_PIN(PORT_D, 3),
+        .sclk_pin = GPIO_PIN(PORT_D, 1),
+        .cs_pin   = SPI_CS_UNDEF,
+        .mosi_af  = GPIO_AF5,
+        .miso_af  = GPIO_AF5,
+        .sclk_af  = GPIO_AF5,
+        .cs_af    = GPIO_AF5,
+        .rccmask  = RCC_APB1ENR1_SPI2EN,
+        .apbbus   = APB1,
+#ifdef MODULE_PERIPH_DMA
+        .tx_dma   = 3,
+        .tx_dma_chan = 1,
+        .rx_dma   = 2,
+        .rx_dma_chan = 1,
+#endif
+    },
```

Comment:
I'm looking at the board user manual and there's no pin assignment for SPI2 on this board. Better remove the configuration for this one.

Reply:
See um2153, table 11, page 37

Reply:
It seems to be on page 39, pins 82,84,85,86, as an alternate function for USART2 on the PMOD. I'd argue we should keep it defined also because removing it would make SPI3 be SPI_DEV(1) instead of 2 and potentially create problems if SPI2 is added back